content/en/docs/concepts/configuration/secret.md

```diff
@@ -896,7 +906,7 @@ spec:
 
 ### Use-case: Dotfiles in secret volume
 
-In order to make piece of data 'hidden' (i.e., in a file whose name begins with a dot character), simply
+In order to make piece of data 'hidden', for example, in a file whose name begins with a dot character, simply
```

Comment:
```suggestion
In order to make piece of data 'hidden' (for example, in a file whose name begins with a dot character),
```

I'm omitting “simply” because it's a bit of a weasel word in documentation, even in this case where a change of filename does look simple.